src/actions/readingStats.js

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import { FETCH_READING_STATS_SUCCESS, FETCH_READING_STATS_REQUEST, FETCH_READING_STATS_FAILED } from '../actionTypes/readingStats';
+
+export const readingStatsRequesting = state => ({
+  type: FETCH_READING_STATS_REQUEST,
+  payload: state,
+})
+
+export const readingStatsError = err => ({
+  type: FETCH_READING_STATS_FAILED,
+  payload: err,
```

Comment:
The payload can be written in full
```suggestion
  payload: error,
```